### ISSUE TITLE
[BUG] Dark theme inactive.

### ISSUE THREAD

**Issue by bayush-9** [Routinger/routinger#17]
**Describe the bug**
The dark theme button in the setting screen doesn't makes any changes.

**To Reproduce**
1. Go to Setting Screen.
2. Toggle the dark screen button.

**Expected behavior**
Turn appearance to dark colors.

**Ways to solve it**
Add code for its implementation.

**Ideas about why is this happening**
It is not implemented yet.

I am wishing to work on it.

**Comment by bayush-9** [comment]
@Kushalrock Let me know the colors you want in dark mode and I will implement it. Please assign me this issue.

**Comment by Kushalrock** [comment]
Yes we added the toggle button. We are yet to work on the dark mode. If you want to work on it. You are more than welcome to do so. All the theme context files can be found in config folder. You can make your desired changes in that file. Also, you may have to change the gradle files. So be careful.

Reply to this comment and I will assign you the task. 

Also, to implement dark mode the colors and stuff you should follow android guidelines or this:
https://www.google.com/amp/s/appinventiv.com/blog/guide-on-designing-dark-mode-for-mobile-app/

**Comment by bayush-9** [comment]
Yeah I'll keep it in mind. In any sort of confusion, I'll ping you here. Also, please add the hactoberfest tag to this issue.

**Comment by Kushalrock** [comment]
Okay I am assigning you to the task. Be sure to check guides on how to implement dark mode in flutter apps and tread accordingly.

Thank you for your contribution.

**Comment by bayush-9** [comment]
#19
